src/cdef.rs

```diff
@@ -643,9 +645,10 @@ mod neon {
         let bottom = bottom.cast();
         let bd = BD::from_c(bitdepth_max);
 
-        let mut tmp_buf = Align16([0; TMP_LEN]);
+        let mut tmp_buf = Align16([MaybeUninit::uninit(); TMP_LEN]);
```

Comment:
```suggestion
        // Use `MaybeUninit` here to avoid over-initialization.
        // C doesn't initialize this either and only partially initializes it in `padding`.
        // Since we're just passing this to a few asm calls that are `unsafe` anyways,
        // initializing this in Rust doesn't really add any extra safety.
        let mut tmp_buf = Align16([MaybeUninit::uninit(); TMP_LEN]);
```